spec/factories/ext_management_system.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
```

Comment:
Honestly, I don't know if that's correct. While working on these changes, I added the `ems_amazon_ebs` in the core repo, but then decided to try adding new factory in the Amazon provider because the EBS manager is really specific to the Amazon provider itself.